### PR TITLE
Removed jQuery usage from admin ( -- now it exists only in views/videos)

### LINKED ISSUE #18975: Remove jQuery usage from project
Remove jQuery usage from project

As we have finished #18756 , now we should remove jQuery from the project too.

## Checklist
- [x] Use Javascript instead of jquery in app/views/admin/comments/index.html.erb
- [x] Use Javascript instead of jquery in app/views/admin/pages/_form.html.erb
- [x] Remove jQuery from app/views/layouts/admin.html.erb
- [ ] Use Javascript instead of jquery in app/views/videos/new.html.erb
- [ ] Remove jQuery from app/views/videos/new.html.erb

### PR DESCRIPTION
**Pull request by rt4914**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This PR removes jQuery usage from admin.

- [x] Use Javascript instead of jquery in app/views/admin/comments/index.html.erb
- [x] Use Javascript instead of jquery in app/views/admin/pages/_form.html.erb
- [x] Remove jQuery from app/views/layouts/admin.html.erb
- [ ] Use Javascript instead of jquery in app/views/videos/new.html.erb -- currently not possible
- [ ] Remove jQuery from app/views/videos/new.html.erb -- currently not possible

## Related Tickets & Documents

- Related to #18975

## QA Instructions, Screenshots, Recordings

I searched for jquery and based on that removed/replaced it wherever poossible.

### UI accessibility concerns?

No

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

File: app/views/videos/new.html.erb [hunk 1]
```diff
@@ -37,8 +37,6 @@
     </p>
   </div>
 </center>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js"
```
Comment (Ridhwana) [inline app/views/videos/new.html.erb:40]: @rt4914 we cannot remove the jQuery from the videos page without making other changes, I can see a usage of jQuery just below https://github.com/forem/forem/blob/76848fa9373231b4174fa910a05acb112566a1a5/app/views/videos/new.html.erb#L45-L48
which will break if we remove jQuery. We'd need to check if there is more like this as well in the other JS files.

Comment (Ridhwana) [reply app/views/videos/new.html.erb:40]: Sorry I know this PR is still in Draft, but I thought I'd bring it to your attention earlier rather than later.

Comment (rt4914) [reply app/views/videos/new.html.erb:40]: Thanks a lot @Ridhwana for this. 
I am actually trying to find the feature where I can actually use this video-upload functionality but unable to find it.

Any idea how to use this feature?

Comment (rt4914) [reply app/views/videos/new.html.erb:40]: Update on above query -- enabled the video setting so as to reach the page which uses jQuery.
Screenshot 2023-01-18 at 4 32 25 PM

Comment (rt4914) [reply app/views/videos/new.html.erb:40]: @Ridhwana I think we cannot remove jQuery. Have a look at this video https://loom.com/share/8786eabfc98f412a9b34aeb367d46a86

Currently we are using `additional_data` [usage](https://github.com/forem/forem/blob/88d9eef916fb80736df3307f034faa4549050a14/app/views/videos/new.html.erb#L46) option which does need jQuery as per https://github.com/waynehoover/s3_direct_upload/blob/master/README.md#options-for-s3upload-jquery-plugin

Comment (rt4914) [reply app/views/videos/new.html.erb:40]: I have tried to find any Javascript/Rails based plugin which does not use jQuery for S3 direct upload but so far no luck in that.

Comment (Ridhwana) [reply app/views/videos/new.html.erb:40]: @rt4914 I think its okay for now, in that case let's keep the jQuery around for the video upload, it's scoped to that page load so I think its okay and we can let @mirie know and perhaps we can make a note of it for the future.  

In the meantime its a big win that we'll be removing it from the admin.html.erb which loads jQuery on all the admin pages. 

Thnak you so much for your effort investigating this!

Comment (rt4914) [reply app/views/videos/new.html.erb]: @Ridhwana Thanks a lot for you guidance. I have removed jquery from admins and updated the code too.

We cannot remove it from app/views/videos/new.html.erb as its is dependent on thrid-party : https://github.com/waynehoover/s3_direct_upload/blob/master/README.md#options-for-s3upload-jquery-plugin

**Comment by rt4914** [comment]
@Ridhwana @fdocr @lboogie04 This PR is ready for review. Thanks.

**Review by Ridhwana — APPROVED** [review]
Tested this and it looks good, thank you for all this @rt4914 ✨

**Review by fdocr — APPROVED** [review]
Also tested locally and LGTM 🙌🏼

**Pull request merged**